src/views/sec/l10n.json

```diff
@@ -1,18 +1,124 @@
+  "sec.org13LinkText": "Hayward Unified School District",
+  "sec.org14": "{org14Listing} - Humble, TX",
+  "sec.org14LinkText": "Humble Independent School District (ISD)",
+  "sec.org15": "{org15Listing} - Guildford, UK",
+  "sec.org15LinkText": "iamtheCODE",
+  "sec.org16": "{org16Listing} - México City, MX",
+  "sec.org16LinkText": "ILCE Instituto Latino Americano de la Comunicación Educativa",
+  "sec.org17": "{org17Listing} - Montréal, Canada",
+  "sec.org17LinkText": "Kids Code Jeunesse",
+  "sec.org18": "{org18Listing} - Clark County School District, NV, USA",
+  "sec.org18LinkText": "Lomie G. Heard Elementary School",
+  "sec.org19": "{org19Listing} - Othello School District, WA, USA",
+  "sec.org19LinkText": "Lutacaga Elementary",
+  "sec.org20": "{org20Listing} - Uganda",
+  "sec.org20LinkText": "Makerere Innovation Society",
+  "sec.org21": "{org21Listing} - NYC, NY, USA",
+  "sec.org21LinkText": "Mouse",
+  "sec.org22": "{org22Listing} - USA",
+  "sec.org22LinkText": "National Society of Black Engineers",
+  "sec.org23": "{org23Listing} - FL, USA",
+  "sec.org23LinkText": "NEFSTEM Center, University of North Florida",
+  "sec.org24": "{org24Listing} - NYC, NY, USA",
+  "sec.org24LinkText": "New York City Department of Education - CSforAllNYC",
+  "sec.org25": "{org25Listing} - NYC, NY, USA",
+  "sec.org25LinkText": "New York Hall of Science",
+  "sec.org26": "{org26Listing} - Nigeria",
+  "sec.org26LinkText": "Odyssey Educational Foundation",
+  "sec.org27": "{org27Listing} - Princeton, NJ, USA",
+  "sec.org27LinkText": "Play Pattern",
+  "sec.org28": "{org28Listing} - Bengaluru, India",
+  "sec.org28LinkText": "Quest Alliance",
+  "sec.org29": "{org29Listing} - Johannesburg, South Africa",
+  "sec.org29LinkText": "Schoolnet South Africa",
+  "sec.org30": "{org30Listing} - Barcelona, Spain",
+  "sec.org30LinkText": "Sistema THEAD",
+  "sec.org31": "{org31Listing} - Phnom Penh, Cambodia",
+  "sec.org31LinkText": "Sisters of Code",
+  "sec.org32": "{org32Listing} - Nairobi, Kenya",
+  "sec.org32LinkText": "STEAMLabs Africa",
+  "sec.org33": "{org33Listing} - Nairobi, Kenya",
+  "sec.org33LinkText": "STEM Impact Center Kenya",
+  "sec.org34": "{org34Listing} - Johannesburg, South Africa",
+  "sec.org34LinkText": "Streetlight Schools",
+  "sec.org35": "{org35Listing} - Chicago Public Schools, IL, USA",
+  "sec.org35LinkText": "Tanner Elementary",
+  "sec.org36": "{org36Listing} - Lagos, Nigeria",
+  "sec.org36LinkText": "Teachathon Foundation",
+  "sec.org37": "{org37Listing} - Girona, Spain",
+  "sec.org37LinkText": "UdiGitalEdu",
+  "sec.org38": "{org38Listing} - Halifax, Nova Scotia",
+  "sec.org38LinkText": "Ulnooweg - Digital Mi'kmaq",
+  "sec.org39": "{org39Listing} - USA",
+  "sec.org39LinkText": "WeTeach_CS, Center for Cyber Education at University of Texas Austin",
+  "sec.org40": "{org40Listing} - Owerri, Nigeria & Nairobi, Kenya",
+  "sec.org40LinkText": "Youth for Technology Foundation",
+  "sec.letterA": "A",
```

Comment:
And those letters; they should not be translated.

Reply:
Just changed